Core/src/Surfaces/LineSurface.cpp

```diff
@@ -239,81 +237,79 @@ Acts::BoundToFreeMatrix Acts::LineSurface::boundToFreeJacobian(
       dDPhiY * boundParams[eBoundLoc0] * ipdn;
   jacToGlobal.block<3, 1>(eFreePos0, eBoundTheta) =
       dDThetaY * boundParams[eBoundLoc0] * ipdn;
+
   return jacToGlobal;
 }
 
 Acts::FreeToPathMatrix Acts::LineSurface::freeToPathDerivative(
     const GeometryContext& gctx, const FreeVector& parameters) const {
   // The global posiiton
-  const auto position = parameters.segment<3>(eFreePos0);
+  Vector3 position = parameters.segment<3>(eFreePos0);
   // The direction
-  const auto direction = parameters.segment<3>(eFreeDir0);
+  Vector3 direction = parameters.segment<3>(eFreeDir0);
   // The vector between position and center
-  const auto pcRowVec = (position - center(gctx)).transpose().eval();
-  // The rotation
-  const auto& rotation = transform(gctx).rotation();
+  Vector3 pcRowVec = position - center(gctx);
   // The local frame z axis
-  const auto& localZAxis = rotation.col(2);
+  Vector3 localZAxis = transform(gctx).rotation() * Vector3::UnitZ();
   // The local z coordinate
-  const auto pz = pcRowVec * localZAxis;
+  double pz = pcRowVec.dot(localZAxis);
   // Cosine of angle between momentum direction and local frame z axis
-  const auto dz = localZAxis.dot(direction);
-  const auto norm = 1 / (1 - dz * dz);
+  double dz = localZAxis.dot(direction);
+  double norm = 1 / (1 - dz * dz);
```

Comment:
Same here, the `const`s don't make a difference, but communicate that these values don't change anymore.

Reply:
consistency